metrics/README.md

```diff
@@ -0,0 +1,168 @@
+      host = "spire-server"
+      port = 8088
+   }
+
+   Statsd = [
+      {
+         address = "graphite-statsd:8125"
+      },
+   ]
+}
+```
+
+The first collector configured is Prometheus. Its configuration accepts two properties, the Prometheus server host which defaults to `localhost` and the Prometheus server port. These values are used by SPIRE to expose an endpoint which will be used by Prometheus to pull the metrics.
+
+For the purpose of this tutorial we configured the host property using the hostname of the SPIRE Server (and Agent) but it is important to note that this configuration allows SPIRE Server and Agent to listen for remote network connections what assumes a risk. When applying a configuration like this in a production environment, the access to the endpoint should be tightly controlled. 
+
+Such scenario generates a warning message on the logs to alert the operator about this risk:
+
+```console
+level=warning msg="Agent is now configured to accept remote network connections for Prometheus stats collection. Please ensure access to this port is tightly controlled." subsystem_name=telemetry
+```
+
+The second collector configured is Statsd. This is one of the collectors that supports the configuration of multiple instances. For that reason the configuration object expects a list of addresses. For this tutorial we define only one instance.
+The address configured matches the Statsd instance running on the environment. We will see the details about this instance in a following section but for now it worth noting that the address is formed by the hostname of the service and the default port for StatsD.
+
+By configuring the address, SPIRE will be pushing metrics to the Statsd collector.
+
+##  Graphite & Statsd Configuration
+
+We use the official Docker image for Graphite & Statsd. This image already contains all the services necessary to collect and display metrics. For this tutorial we map the port `80` that belongs to the nginx proxy that reverse proxies the Graphite dashboard and the port `8125` where Statsd listens by default.
+The `graphite-statsd` service definition is:
+
+```console
+  graphite-statsd:
+    image: graphiteapp/graphite-statsd:1.1.7-6
+    container_name: graphite
+    hostname: graphite-statsd
+    restart: always
+    ports:
+        - "80:80"
+        - "8125:8125/udp"
+```
+
+The Statsd service will be available at `graphite-statsd:8125` as configured for SPIRE Server in the previous section.
+
+## Prometheus Configuration
+
+Due to the pull nature of Prometheus we need to configure the HTTP endpoint where it will scrape the metrics. We've already configured SPIRE to expose the HTTP endpoint via the telemetry configuration so now we need to indicate to Prometheus that it should collect metrics from that endpoint. We achieve this by configuring the target with the hostname of the SPIRE server (or SPIRE Agent) and the correct port number (e.g. 8088 for the SPIRE Server and 8089 for the SPIRE Agent).
```

Comment:
```suggestion
Due to the pull nature of Prometheus we need to configure the HTTP endpoint where it will scrape the metrics. We've already configured SPIRE to expose the HTTP endpoint via the telemetry configuration so now we need to indicate to Prometheus that it should collect metrics from that endpoint. We achieve this by configuring the `target` setting with the hostname of the SPIRE server (or SPIRE Agent) and the correct port number (e.g. 8088 for the SPIRE Server and 8089 for the SPIRE Agent).
```

Reply:
Here, "with" is needed. However, it might be better to rephrase it using "to." -->

We achieve this by setting the `target` option to the hostname of the SPIRE server (or SPIRE Agent) and the correct port number (e.g. 8088 for the SPIRE Server and 8089 for the SPIRE Agent).

Reply:
I like that. I'll change it using the new suggestion. Thanks